llvm/lib/CodeGen/AsmPrinter/AsmPrinter.cpp

```diff
@@ -161,6 +161,12 @@ static cl::bits<PGOMapFeaturesEnum> PgoAnalysisMapFeatures(
         "Enable extended information within the SHT_LLVM_BB_ADDR_MAP that is "
         "extracted from PGO related analysis."));
 
+static cl::opt<bool>
+    SkipEmitBBEntries("skip-emit-bb-entries",
```

Comment:
I suggest `basic-block-address-map-skip-bb-entries` to spell out the relationship to the feature. Also please add the motivation in the description (e.g., binary size savings).

Reply:
Updated, thanks for the suggestion!